Extensions/BBText/JsExtension.js

```diff
@@ -520,10 +520,7 @@ module.exports = {
         .getProperties(this.project)
         .get('fontFamily')
         .getValue();
-      this._pixiObject.textStyles.default.fontFamily = this._pixiResourcesLoader.getFontFamily(
-        this.project,
-        fontFamily
-      );
+      this._pixiObject.textStyles.default.fontFamily = this._pixiResourcesLoader.loadFontFamily(this.project, fontFamily);
```

Comment:
`loadFontFamily` returns a promise containing the result of the loading, which is the name of the font family to be used. You can't affect it to font family, this won't work (you're affecting a Promise, which is an object, to a string).

Instead, look at how it's done in: https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/ObjectsRendering/Renderers/RenderedTextInstance.js

The reason for having a promise is because the IDE has to load the font - it can't return to you the font family immediately.